### PR TITLE
Update biofeedback patch

### PR DESCRIPTION
**Pull request by JanCBrammer**
### module
* added polarbelt module
* added breathingbiofeedback module
* minor changes in recordtrigger module

### patches
* updated biofeedback patch

### lib
* removed unused imports in EEGsynth.py
* added Bessel filters to EEGsynth.py

### CODE REVIEW

File: module/polarbelt/polarbelt.py [hunk 1]
```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env python3
```
Comment (robertoostenveld) [inline module/polarbelt/polarbelt.py:1]: `/usr/bin/env python` should result in python3 being executed

File: module/recordtrigger/recordtrigger.py [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
```
Comment (robertoostenveld) [inline module/recordtrigger/recordtrigger.py:1]: `/usr/bin/env python` should result in python3 being executed

File: module/recordtrigger/recordtrigger.py [hunk 3]
```diff
@@ -99,6 +100,7 @@ def _setup():
     args = parser.parse_args()
 
     config = configparser.ConfigParser(inline_comment_prefixes=('#', ';'))
+    config.optionxform = str
```
Comment (robertoostenveld) [inline module/recordtrigger/recordtrigger.py:103]: what does this do?

File: module/polarbelt/polarbelt.py (hunk 1 above)
Comment (robertoostenveld) [reply module/polarbelt/polarbelt.py:1]: can you change this in `python` or provide arguments why python3 is required? If python3 is required, would it also apply elsewhere?

Comment (robertoostenveld) [reply module/polarbelt/polarbelt.py:1]: oh, not that I mean that the code should run with python2, but just that the executable name should be python rather that pythonX with X being the version. If the code (as in this case) has requirements on the Python version, please add them to the start of the code using `sys.version_info`. See for example `inputosc.py`

File: module/recordtrigger/recordtrigger.py (hunk 3 above)
Comment (JanCBrammer) [reply module/recordtrigger/recordtrigger.py:103]: It's [preserves the case (upper, lower)](https://stackoverflow.com/questions/1611799/preserve-case-in-configparser) in the Redis keys (we use  CamelCase for the biochill patch).

File: module/recordtrigger/recordtrigger.py (hunk 2 above)
Comment (JanCBrammer) [reply module/recordtrigger/recordtrigger.py:1]: Indeed. I think this is left over from before I made Python 3 the default Python on the Pi. Will change it.

File: module/polarbelt/polarbelt.py (hunk 1 above)
Comment (JanCBrammer) [reply module/polarbelt/polarbelt.py:1]: See my other comment below.

**Comment by JanCBrammer** [comment]
> the name "biofeedback" as a patch is not specific enough, since all patches have to do with biofeedback somehow. I suggest to keep it as "biochill", since that is a name that people can google

Makes sense, will rename both the patch and the `breathingbiofeedback` module to `biochill`.

**Comment by robertoostenveld** [comment]
Looks good to me. @JanCBrammer is this ready to merge?

**Comment by JanCBrammer** [comment]
> is this ready to merge?

Yes, tested the latest version of the biochill patch earlier today (after adding plotcontrol), and I don't forsee major changes to the patch in the coming month.

**Pull request merged**